data/successors/successor 1 introduction.txt

```diff
@@ -0,0 +1,1085 @@
+				goto released
+			label successful
+			`	With the larger animals dispatched, you and Sieasej dash into your ship with your quarry safe in tow. The waves of color running over Sieasej's body slowly fade. "Well done, captain," they eventually say. "I'm sure Sioeora will be pleased."`
+			`	While Sieasej returns to their cabin, you store the captured Kijra-riitaa in a spare animal-transport cargo container in the <ship>'s hold. It seems to dislike being caged, growling gently and pacing the container, kicking up bits of the straw inside.`
+			choice
+				`	(Leave it be.)`
+					goto takeoff
+				`	(Put some more bedding in the container.)`
+				`	(Release it.)`
+					goto betterlatethannever
+			action
+				set "Successors FC3: captured kijra-pet"
+			`	You rummage around in the other cargo containers until you find a bit of spare fabric scraps and leaf litter, a couple of handfuls of which you add to the animal's cage. With that sorted out, you make your preparations for takeoff.`
+				goto end
+			label takeoff
+			action
+				set "Successors FC3: captured kijra-pet"
+			`	With the animal stowed away, you make your preparations for takeoff.`
+				goto end
+			label betterlatethannever
+			action
+				set "Successors FC3: released kijra-pet"
+			`	You open the container's door and coax the Kijra-riitaa out of your ship. It takes flight shortly after stepping outside, soon disappearing into a speck against the vast blue sky. With nothing left to do here, you make your preparations for takeoff so that you can at least return Sieasej to their people.`
+				goto end
+			label released
+			`	The waves of color running over Sieasej's body slowly fade once the animals are gone. "I confess I hadn't picked thee for such a tender soul, captain," they eventually say.`
+			choice
+				`	"I only did what I thought would get us back alive."`
+					goto backalive
+				`	"It wasn't right to hurt them."`
+			`	"Ah," they say. "We are fortunate they thought the same of us."`
+				goto released-2
+			label backalive
+			`	"Ah," they say. "Well I, for one, shall thank thee for thy prudence." After a delay, they add, "Even if Sioeora will not."`
+			label released-2
+			`	Sieasej briefly waves their device in the air unsuccessfully. "Nonetheless," they say, "thine act of mercy entails us failure. Under threat, the adults shall surely shroud the larvae, and so it is now here. It seems we must return with empty arms." They briefly undulate in a manner similar to a laugh. "I do not think I'd take another chase like that again regardless."`
+			`	The two of you return to the <ship> and make preparations for takeoff so that you can, at the very least, return Sieasej to their people.`
+			label end
+
+	on fail
+		dialog `You failed to return Sieasej to their people in time. If you wish to continue the Successors' story, revert to a prior save.`
+		"reputation: House Sioeora" = -999
```

Comment:
-999 was just the placeholder value I chose to indicate that they hate you permanently. There's a similar thing where the Gegno Genocide Defense mission sets all Gegno reps to -1000; I suppose I just found -999 stood out more against the rest.

That being said, it probably makes sense to change the People's Houses (Hostile) government to use a less extreme negative value so that the player could conceivably work their way into their good graces.